File: win-prog/ProgWinVB/Toolbars and Status Bars/TwoStatusBarPanels/TwoStatusBarPanels.vb
'---------------------------------------------------
' TwoStatusBarPanels.vb (c) 2002 by Charles Petzold
'---------------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Class TwoStatusBarPanels
    Inherits Form

    Shared Sub Main()
        Application.Run(new TwoStatusBarPanels())
    End Sub

    Sub New()
        Text = "Two Status Bar Panels"
        BackColor = SystemColors.Window
        ForeColor = SystemColors.WindowText

        Dim sbar As New StatusBar()
        sbar.Parent = Me
        sbar.ShowPanels = True

        Dim sbp1 As New StatusBarPanel()
        sbp1.Text = "Panel 1"

        Dim sbp2 As New StatusBarPanel()
        sbp2.Text = "Panel 2"

        sbar.Panels.Add(sbp1)
        sbar.Panels.Add(sbp2)
    End Sub
End Class

File: win-prog/ProgWinCS/Toolbars and Status Bars/TwoStatusBarPanels/TwoStatusBarPanels.cs
//-------------------------------------------------
// TwoStatusBarPanels.cs © 2001 by Charles Petzold
//-------------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class TwoStatusBarPanels: Form
{
     public static void Main()
     {
          Application.Run(new TwoStatusBarPanels());
     }
     public TwoStatusBarPanels()
     {
          Text = "Two Status Bar Panels";
          BackColor = SystemColors.Window;
          ForeColor = SystemColors.WindowText;

          StatusBar sb = new StatusBar();
          sb.Parent = this;
          sb.ShowPanels = true;

          StatusBarPanel sbpanel1 = new StatusBarPanel();
          sbpanel1.Text = "Panel 1";

          StatusBarPanel sbpanel2 = new StatusBarPanel();
          sbpanel2.Text = "Panel 2";

          sb.Panels.Add(sbpanel1);
          sb.Panels.Add(sbpanel2);
     }
}
